lib/galaxy/metadata/__init__.py

```diff
@@ -17,7 +17,24 @@
 
 log = getLogger(__name__)
 
-SET_METADATA_SCRIPT = "from galaxy_ext.metadata.set_metadata import set_metadata; set_metadata()"
+SET_METADATA_SCRIPT = """
+import os
```

Comment:
Note that I tried not to import anything from galaxy, so that the error handling works even if there's something wrong with the python environment